rules/expiring-todo-comments.js

```diff
@@ -17,7 +17,8 @@ const MESSAGE_ID_ENGINE_MATCHES = 'engineMatches';
 const MESSAGE_ID_REMOVE_WHITESPACES = 'removeWhitespaces';
 const MESSAGE_ID_MISSING_AT_SYMBOL = 'missingAtSymbol';
 
-const pkg = readPkgUp.sync().package;
+const hasPackage = readPkgUp.sync();
+const pkg = hasPackage ? hasPackage.package : {};
```

Comment:
This two line seems weird.

Reply:
first thinking

```js
const {package: pkg = {}} = readPkgUp.sync() || {};
const hasPackage = ??? // can't detect
```

next might better, but extra line

```js
const result= readPkgUp.sync();
const hasPackage = Boolean(result)
const pkg = hasPackage ? result.package : {};
````

or maybe this one

```js
const {
  package: pkg = {}, 
  hasPackage = true
} = readPkgUp.sync() || {hasPackage: false}
```

@lubien what do you think

Reply:
I agree. I just needed to get this out.

Should probably have been:

```js
const packageResult = readPkgUp.sync();
const hasPackage = Boolean(packageResult);
const packageJson = hasPackage ? packageResult.package : {};
```

Reply:
Did we comment at the same time? LOL
what do you think my last code

Reply:
Hehe. Seems so. I didn't see your comment until now. Let's go with mine, which is your second one, just with more verbose naming.

Reply:
/cc @lubien Could you please send another PR to apply this?

Reply:
Sure! But I need a few hours before I can do this if it's not an issue.

Reply:
https://github.com/sindresorhus/eslint-plugin-unicorn/pull/399